Portscout/SiteHandler/GitHub.pm

```diff
@@ -141,20 +125,9 @@ sub GetFiles
 
 	# In case there aren't any releases, try tags tags instead
 	if (scalar @$files == $files_count_before) {
-		$query = 'https://api.github.com/repos/' . $projname . '/tags' . $credentials;
-		_debug("GET $query");
-		$ua = LWP::UserAgent->new;
-		$ua->agent(USER_AGENT);
-		$ua->timeout($settings{http_timeout});
-
-		$response = $ua->request(HTTP::Request->new(GET => $query));
-
-		if (!$response->is_success || $response->status_line !~ /^2/) {
-			_debug('GET failed: ' . $response->status_line);
-			return 0;
-		}
-		$json = decode_json($response->decoded_content);
-		foreach my $tag (@$json) {
+		my $tags = _call_github_api('/repos/' . $projname . '/tags')
+		  or return 0;
+		foreach my $tag (@$tags) {
```

Comment:
Same as above (thanks to having the function call, we won't have to repeat everything).